app/src/main/java/com/example/musketeerbubblesort/MainActivity.java

```diff
@@ -21,13 +27,55 @@ public void QuitApp(View view) {
         this.finishAffinity();
     }
 
+    @RequiresApi(api = Build.VERSION_CODES.N)
     public void sortInputDataFromUser(View view) {
         Intent intent = new Intent(this, DisplaySortedInformationActivity.class);
         EditText editText = (EditText) findViewById(R.id.userInput);
+        Button submit = (Button) findViewById(R.id.button);
         String message = editText.getText().toString();
-        intent.putExtra(DATA_INPUT_MESSAGE, message);
-        startActivity(intent);
-    }
-
+        int[] unsortedInput = Arrays.stream(message.split(" ")).mapToInt(Integer::parseInt).toArray();
+        /*
+         *  if statements that will output a user error when the array inputted are not not single
+         * integer between the size of 3 and 8
+         * @param unsortedInput The user input in the form of an array of integers
```

Comment:
This is just a little oversight here, you previously added a method here and when you removed the method you left the comments.